docs/howto/support/raise-issues.md

```diff
@@ -1,124 +1,139 @@
+7. Click **Create** to submit your ticket.
 
-These request types are for requests relating to our Managed Services:
+> **Tip**: Use the **search bar** at the top of the portal page to quickly find your tickets or search our knowledge base for helpful articles.
 
-- File Backup
-- Hosting
+---
 
-![Managed Services Category and Issue Types](./assets/03-managed-services.png 'Managed Services Category and Issue Types showing the labels "File Backup" with the description "Create, Cancel or Restore client for Managed File Backup" and "Hosting" with the description "Create or Cancel server for Managed Hosting"')
+## Raising Tickets via Email
 
-## Ticket communication
+If you choose to raise a ticket by email, please follow these guidelines:
 
-So now you have created your issue ticket, what's next?
+1. **Email Address**: Send your email to [{{support_email}}](mailto:{{support_email}}) from an account [matching your profile information](../account-billing/change-account-data.md).
+2. **Language**: Use English for quicker responses. While some staff speak other languages, using languages other than English may delay responses.
+3. **Subject**: Provide a clear and descriptive subject line, e.g., "Request to increase project quota" or "Unable to log in to {{gui}}."
+4. **Details**: Include as much information as possible, especially:
+     - Steps you’ve taken to resolve the issue.
+     - The UUID (Universally Unique Identifier) of affected resources, if applicable.
+
+> **Note:** Customers with a paid support package must use the **{{support}} Portal** for raising tickets.
+
+---
 
-You should have received an email notification with a confirmation that issue your ticket has been created, if you want to provide additional details you can reply to this email.
-One of our Technical Support Engineers will soon get assigned to your case, confirm the details, and then take the proper actions to fulfill or resolve the request or issue, which will be communicated.
+## Ticket Communication
 
-Be prepared that our Support Engineers may request additional information from you in order to resolve the issue.
+Once your ticket is created, here’s what to expect:
 
-If you have a Support Package subscription, you should always use the {{support}} portal for further communication, this helps us authenticate you.
-This also makes it easier for you to track the communication and status of your issue ticket(s).
+1. **Confirmation Email**: You will receive an email confirming your ticket submission. To add more details, reply directly to this email.
+2. **Support Engineer Assignment**: A Technical Support Engineer will review your case, confirm the details, and take the necessary steps to resolve or fulfill your request.  
+3. **Additional Information Requests**: Be prepared to provide additional details if requested by the Support Engineer.
```

Comment:
Is There A Specific Reason For Using Title Case: for anything that's before the `:`?

Reply:
No reason. Just some weird reflex I have.